common/sgx/collateral.c

```diff
@@ -43,6 +43,20 @@ oe_result_t __oe_sgx_set_minimum_crl_tcb_issue_date(
     return result;
 }
 
+static oe_result_t _get_crl_ca_type(
+    uint8_t* platform_instance_id,
+    oe_get_sgx_quote_verification_collateral_args_t* args)
+{
+    uint8_t null_platform_id[16] = {0};
+    if (memcmp(
+            platform_instance_id, null_platform_id, sizeof(null_platform_id)) ==
```

Comment:
It's not immediately obvious locally that platform_instance_id is at least 16 bytes long.

Reply:
Thanks. I'll add a comment that platform_instance_id here is one of SGX PCK certificate extension fields which is 16 bytes long